python/lsst/ctrl/bps/transform.py

```diff
@@ -231,6 +216,8 @@ def _enhance_command(config, generic_workflow, gwjob):
     gwjob : `lsst.ctrl.bps.GenericWorkflowJob`
         Generic workflow job to which the updated executable, arguments,
         and values should be saved.
+    cached_job_values : `dict` [`str`, `str`]
```

Comment:
Shouldn't the type description for `cached_job_values` be `dict [dict [str, Any]]`? Unless I got it wrong, it looks like it's a nested dictionary.

Reply:
Also, I'd appreciate mentioning in `cached_jobs_values` description that it will be modified by the function.